awebox/pmpc.py

```diff
@@ -261,7 +272,7 @@ def __extract_solver_stats(self, sol):
         """
 
         info = self.__solver.stats()
-        self.__log['cpu'].append(info['t_wall_solver'])
+        # self.__log['cpu'].append(info['t_wall_total'])
```

Comment:
why did you comment this out without a replacement?

Reply:
oh yes, I guess it should be uncommented, but this will become obsolete when bumping to casadi 3.5 anyway. For now, I'll put it back in.